clang/test/Analysis/stream.c

```diff
@@ -65,12 +65,24 @@ void check_fseek(void) {
   fclose(fp);
 }
 
+void check_fseeko(void) {
```

Comment:
If we have these null pointer tests for all functions, `vfprintf` and `vfscanf` could be added too.

Reply:
Added